http/logical.go

```diff
@@ -29,15 +29,20 @@ func handleLogical(core *vault.Core, dataOnly bool) http.Handler {
 		case "DELETE":
 			op = logical.DeleteOperation
 		case "GET":
+			// Need to call ParseForm to get query params loaded
+			err := r.ParseForm()
```

Comment:
`req.URL.Query()` is better, since ParseForm will also parse a form body which is not technically part of the specification.